results-processor/README.md

```diff
@@ -0,0 +1,28 @@
+## Basics
+
+The results processor runs on Python 3.6. The entry point is a Flask web server
+(`main.py`). In production, gunicorn is used as the WSGI (see `Dockerfile`) and
+the container runs as a custom AppEnging Flex instance (see `app.yaml`).
+
+## Getting started
+
+We can create a virtualenv to recreate a setup close to production for daily
+development.
+
+```bash
+virtualenv env -p python3.6
+. env/bin/activate
+pip install -r requirements.txt
+```
+
+## Running tests
+
+We strongly recommend you to run tests **outside of** the virtualenv created
```

Comment:
By just reading these docs I don't understand where the nested virtualenvs are. There's just one activate step described above.

Reply:
`tox` will automatically create one, so if you're still in the virtualenv created above, running tox gives you a nested virtualenv, which is usually fine but occasionally you run into weird PATH issues.

Reply:
Oh right, tox.